packages/core-data/src/resolvers.js

```diff
@@ -118,3 +119,18 @@ export function* hasUploadPermissions() {
 
 	yield receiveUploadPermissions( includes( allowHeader, 'POST' ) );
 }
+
+/**
+ * Request autosave data from the REST API.
+ *
+ * @param {Object} post The post that is the parent of the autosave.
```

Comment:
Question: How much more difficult would it be to implement as accepting the post's ID, not the object? The former is much more accessible, thus making the selector easier to use. Generally I think we should seek to accept simpler arguments. While not relevant here, it also enables easier function memoization, since complex arguments can't be compared using strict equality conditions.

Reply:
The post's type is needed to form the correct path to the rest endpoint (wp/v2/:post_type_rest_base/:id/autosaves). I can change the signature to `getAutosave( postType, postId )`. Not necessarily easier for the implementor, but at least simpler types.

Reply:
Hm. It's a shame that despite post IDs being unique across all post types, there's no way to retrieve a post from the API knowing only its ID.

I still think the variant you suggest would be preferable, though the ultimate ideal for me would be one which accepted just the ID.

Reply:
Posed the question in the `core-restapi` Slack channel:

https://wordpress.slack.com/archives/C02RQC26G/p1546983215113700

I don't think there's currently a way to get the entity with only the ID. There was some contemplation as to whether this could be an addition to the search endpoint.

Reply:
The other thing that adds complexity is that the autosave is fetched by its parent's postType and postId rather than its own postId, so that would make it quite a specialised endpoint.